packages/components/src/List/Toolbar/ColumnChooser/hooks/columnChooserClient.hook.js

```diff
@@ -0,0 +1,20 @@
+import { useState, useEffect } from 'react';
+import { mergedColumnsChooser } from '../service';
+
+function useColumnChooserClient(columns, handlerCustomColumnChooser = () => {}) {
+	const [state, setState] = useState({ columns: columns || [] });
```

Comment:
idk but what do you think of 
```
const [columns, setColumns] = useState(columns)
```
and then in the code beside =>
```
setColumns(editedColumns);
```

Reply:
In custom hooks that have only one state but multiple field (like the old state), I think have a state / setState can be more readable.
I choose to have an object as a state to give flexibility.
If I only initialize it with colums, the state will be literally the columns.